dcs/src/main/java/org/trafodion/dcs/server/ServerManager.java

```diff
@@ -255,33 +259,17 @@ private void cleanupZk() {
         CountDownLatch startSignal = new CountDownLatch(1);
         RetryCounter retryCounter;
 
-        public void reset() {
-            startSignal.countDown();
-            startSignal = new CountDownLatch(1);
-            boolean isRunning = this.serverMonitor.monitor();
-            String nid = this.serverMonitor.nid;
-            String pid = this.serverMonitor.pid;
-
-            if (isRunning) {
-                LOG.info("mxosrvr " + nid + "," + pid + " still running");
-                this.retryCounter.resetAttemptTimes();
-            } else {
-                LOG.info("mxosrvr " + nid + "," + pid + " exited, restarting, restart attempt time : "
-                        + this.retryCounter.getAttemptTimes());
-            }
-        }
-
         public ServerHandler(Configuration conf ,int childInstance) {
             int maxRestartAttempts = conf.getInt(Constants.DCS_SERVER_USER_PROGRAM_RESTART_HANDLER_ATTEMPTS,
                     Constants.DEFAULT_DCS_SERVER_USER_PROGRAM_RESTART_HANDLER_ATTEMPTS);
-            int retryIntervalMillis = conf.getInt(
-                    Constants.DCS_SERVER_USER_PROGRAM_RESTART_HANDLER_RETRY_INTERVAL_MILLIS,
-                    Constants.DEFAULT_DCS_SERVER_USER_PROGRAM_RESTART_HANDLER_RETRY_INTERVAL_MILLIS);
+            int retryIntervalMinutes = conf.getInt(
```

Comment:
Why did we change from millis to minutes. Millis is more granular and you can achieve mins from millis